project/program/templates/program/speaker.html

```diff
@@ -0,0 +1,25 @@
+{% extends "base.html" %}
+{% block content %}
+<article>
+    <button class="btn btn-secondary"> < Back to speakers</button>
+    <div class="speaker-main">
+        <div class="speaker-image"> </div>
```

Comment:
μπορούμε να βάλουμε την πραγματική εικόνα πλέον

<details>
<summary>Just show me the code</summary>
<a href="https://github.com/tdiam/tedxntua2020/blob/event_pages/project/program/templates/program/listing.html">εδώ</a> το κομμάτι με το <code>&lt;img&gt;</code>
</details>
<details>
<summary>Πιο αναλυτικά</summary>
<a href="https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images">How do you create responsive images?</a>
</details>